lib/mongo/crypt/handle.rb

```diff
@@ -22,23 +22,28 @@ module Crypt
     # allowing clients to set options on that object or perform operations such
     # as encryption and decryption
     class Handle
-
       # Creates a new Handle object and initializes it with options
       #
       # @param [ Hash ] kms_providers A hash of KMS settings. The only supported key
       # is currently :local. Local KMS options must be passed in the format
       # { local: { key: <master key> } } where the master key is a 96-byte, base64
       # encoded string.
+      # @param [ Hash ] options A hash of options
+      #
+      # @option [ Logger ] :logger A Logger object to which libmongocrypt logs
+      #   will be sent
       #
       # There will be more arguemnts to this method once automatic encryption is introduced.
-      def initialize(kms_providers)
+      def initialize(kms_providers, options={})
         # FFI::AutoPointer uses a custom release strategy to automatically free
         # the pointer once this object goes out of scope
+        @logger = options[:logger]
```

Comment:
@p-mongo I changed the API to accept a logger as an option, with the intention of passing it down from the Client or ClientEncryption object. Does that seem reasonable?